packages/create-playwright/assets/playwright.config.js

```diff
@@ -18,10 +18,10 @@ const config = {
 
   // Run your local dev server before starting the tests:
   // https://playwright.dev/docs/test-advanced#launching-a-development-web-server-during-the-tests
-  // webServer: {
-  //   command: 'npm run start',
-  //   port: 3000,
-  // },
+  webServer: {
+    command: 'npx -y playwright-examples',
```

Comment:
I'd just include source in the examples, and do `node my-server.js`. Or use `npx serve` to server a static site.